example/rails-51/config/routes.rb

```diff
@@ -1,9 +1,26 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
```

Comment:
Are these docs now obsolete, or is there another reason why we no longer link to them?

Reply:
It's just scaffold comments from the generation of the project, I removed it as it seems irrelevant.